src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -272,14 +272,15 @@ public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
             .getEpimsIdForVenue(caseData.getProcessingVenue())
             .orElse(null);
 
-        HearingLocation hearingLocation = new HearingLocation();
-        hearingLocation.setLocationId(epimsId);
-        hearingLocation.setLocationType(COURT);
-
-        List<HearingLocation> hearingLocations = new ArrayList<>();
-        hearingLocations.add(hearingLocation);
+        Resource resource = new ClassPathResource("multipleHearingLocations.json");
```

Comment:
Why is this a concurrent hashmap?

Reply:
Possibly for the future: we might want to consider moving the load into a separate bean so we don't parse the file on every request. @filippo-possenti-version1 @ss-evoco

Reply:
I had to implement it to get rid of PMD supressions

Reply:
Agreed for the future. Let's unblock this and we put all these improvements up as technical debt.

Reply:
With regards to: 

`        ConcurrentHashMap<String,List<String>> multipleHearingLocations = new ObjectMapper().readValue(Files.newInputStream(Paths.get(resource.getURI())), new TypeReference<ConcurrentHashMap<String, List<String>>>(){});
`

Could you please let me know what the PMD error is? Did you make sure both the TypeReference and TypeDefinition matched?

Loading the resource can be simplified to something like: https://www.baeldung.com/jackson-object-mapper-tutorial (see section 3.2) you shouldn't need to load the resource, then use paths and create a new input stream.

It's not the best solution but ReferenceDataServiceHolder is used as a placeholder to pass beans around. I would recommend loading the json file in that class so it's loaded once into memory. That way you should be able to simply get the map when you need it instead of loading on every request.

Reply:
Ok, so basically in the ReferenceDataService holder we add the following:
```
@Getter(lazy=true)
private final Map<String, List<String>> multipleHearingLocations = prepareMultipleHearingLocations();

private Map<String, List<String>> prepareMultipleHearingLocations() {
    return ImmutableMap.copyOf(OBJECT_MAPPER.readValue(new ClassPathResource("multipleHearingLocations.json").getInputStream(), new TypeReference<Map<String, List<String>>>(){}));
}
```

Then the line in this class can be turned into something like this:
```
Map<String,List<String>> multipleHearingLocations = referenceDataServiceHolder.getMultipleHearingLocations();
```

Then all we have left is to add this line just above the method's signature:
```
@SuppressWarnings({"PMD.UseConcurrentHashMap"})
```

To get rid of the false-positive on ConcurrentHashMap (if it's still needed) and we should be good.